.github/workflows/master.yml

```diff
@@ -24,10 +24,6 @@ jobs:
           role-to-assume: "arn:aws:iam::292999226676:role/github_actions-test-infra-cluster"
           aws-region: eu-west-1
           
-      - name: Deploy terraform
```

Comment:
Also, we can cleanup the script, right?
Moreover, i'd rename this CI to `Master CI - deploy prow` and the new one to `Master CI - apply terraform`.

Reply:
Totally agree @FedeDP. Implemented, PTAL